test/runtests.jl

```diff
@@ -160,6 +161,8 @@ AstronomicalTime.update()
         @test AstronomicalTime.Epochs.ut1tai(julian2(ut1), julian1(ut1), dat_ut1) == ERFA.ut1tai(julian2(ut1), julian1(ut1), dat_ut1)
         @test AstronomicalTime.Epochs.taiut1(julian1(tai), julian2(tai), dat_tai) == ERFA.taiut1(julian1(tai), julian2(tai), dat_tai)
         @test AstronomicalTime.Epochs.taiut1(julian2(tai), julian1(tai), dat_tai) == ERFA.taiut1(julian2(tai), julian1(tai), dat_tai)
+        @test +(AstronomicalTime.Epochs.tcgtt(julian1(tcg), julian2(tcg))...) ≈ +(ERFA.tcgtt(julian1(tcg), julian2(tcg))...)
+        @test +(AstronomicalTime.Epochs.tcgtt(julian2(tcg), julian1(tcg))...) ≈ +(ERFA.tcgtt(julian2(tcg), julian1(tcg))...)
```

Comment:
A simpler way would be:
```julia
sum(AstronomicalTime.Epochs.tcgtt(julian1(tcg), julian2(tcg))) ≈ sum(ERFA.tcgtt(julian1(tcg), julian2(tcg)))
```

Did you investigate why the stricter test (`==`) is not passing?

Reply:
Yes, it's due to the error in floating point precision.

Reply:
tcg = Epoch{TCG}(2.4578265e6, 0.30440190993249416)
>2017-03-14T07:18:20.325 TCG

Reply:
Can you give me an example?

Reply:
julia> ERFA.tcgtt(julian1(tcg), julian2(tcg))
(2.4578265e6, 0.3043916774088325)

julia> AstronomicalTime.Epochs.tcgtt(julian1(tcg), julian2(tcg))
(2.4578265e6, 0.3027549702857392)

julia> tcg = Epoch{TCG}(2.4578265e6, 0.30440190993249416)
2017-03-14T07:18:20.325 TCG

julia> JD = ERFA.tcgtt(julian1(tcg), julian2(tcg))
(2.4578265e6, 0.3043916774088325)

julia> TCGEpoch(JD...)
2017-03-14T07:18:19.441 TCG